packages/palette/src/Theme.tsx

```diff
@@ -48,6 +48,25 @@ export const unitlessBreakpoints = {
   xs: parseInt(breakpoints.xs, 10),
 }
 
+const BREAKPOINTS_SCALE = [
+  breakpoints.sm,
+  breakpoints.md,
+  breakpoints.lg,
+  breakpoints.xl,
+]
+
+// Alias these values so that we can use object syntax in props
+// @ts-ignore
+BREAKPOINTS_SCALE.xs = breakpoints.xs
+// @ts-ignore
+BREAKPOINTS_SCALE.sm = breakpoints.sm
+// @ts-ignore
+BREAKPOINTS_SCALE.md = breakpoints.md
+// @ts-ignore
+BREAKPOINTS_SCALE.lg = breakpoints.lg
+// @ts-ignore
+BREAKPOINTS_SCALE.xl = breakpoints.xl
```

Comment:
styled-system requires that you stick these aliases directly on the scale object.

Reply:
It's a bit of a head trip to see an array tacking on object props -- can we use an intermediate variable so that `BREAKPOINTS_SCALE` can remain consistent type-wise at runtime?

Reply:
This was a good callout. For whatever reason I didn't think this was possible but simple `Object.assign` here works perfectly

![](http://static.damonzucconi.com/_capture/ZFB3Gnf9BvqD.png)